File: Manual 02/Question08.cpp
#include <iostream>
using namespace std;

int main()
{
    int x;
    cout << "Input Year: ";
    cin >> x;

    if (x % 4 == 0)
    {
        if (x % 400 == 0)
        {
            cout << "It is not a leap year!";
        }
        else
        {
            cout << "It is a Leap Year!";
        }
    }
    if (x % 4 != 0)
    {
        cout << "It is not a leap Year!";
    }
}

File: Python/Manual 02/Question08.py
year = int(input("Enter Year: "))

if year < 0:
    print("Enter a Valid Year")
else:
    if year % 100 == 0 or year % 4 != 0:
        print("Not a Leap Year")
    elif year % 4 == 0:
        print("Leap Year")
